tests/xhtml/TestChunkedXHTML.php

```diff
@@ -0,0 +1,39 @@
+<?php
+namespace phpdotnet\phd;
+
+class TestChunkedXHTML extends Package_Generic_ChunkedXHTML {
+    public function update($event, $val = null) {
+        switch($event) {
+        case Render::CHUNK:
+            parent::update($event, $val);
+            break;
+        case Render::STANDALONE:
+            parent::update($event, $val);
+            break;
+        case Render::INIT:
+            $this->setOutputDir(Config::output_dir() . strtolower($this->getFormatName()) . '/');
+            break;
+        //No verbose
+        }
+    }
+
+    public function writeChunk($id, $fp) {
+        $filename = $this->getOutputDir() . $id . $this->getExt();
+
+        rewind($fp);
+        $content = "\n";
+        $content .= stream_get_contents($fp);
+
+        if ($id === "") {
+            $filename = Config::xml_file();
+        }
+
+        echo "Filename: " . basename($filename) . "\n";
+        echo "Content:" . $content . "\n";
+    }
+}
+
+/*
+* vim600: sw=4 ts=4 syntax=php et
+* vim<600: sw=4 ts=4
+*/
```

Comment:
Drop this maybe?

Reply:
Just about every files has these lines... Maybe a follow up PR removing all of these at once?

Reply:
Possibly a good idea yes, well follow-up it is

Reply:
Done.